examples/vector_databases/supabase/llamaindex/llamaindex.ipynb

```diff
@@ -0,0 +1,189 @@
+{
+  "cells": [
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "id": "4f3b89c3",
+      "metadata": {},
+      "source": [
+        "[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/supabase/supabase/blob/master/examples/ai/llamaindex/llamaindex.ipynb)"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "id": "2a1f181d-feeb-4b29-aabc-67a75234b92c",
+      "metadata": {},
+      "source": [
+        "# Supabase + LlamaIndex\n",
+        "\n",
+        "In this example we'll use PostgreSQL + pgvectors with [LlamaIndex's Supabase Vector Store](https://gpt-index.readthedocs.io/en/latest/examples/vector_stores/SupabaseVectorIndexDemo.html#setup-openai).\n"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "id": "3cab93f5-10d0-47c5-9f4e-64921461e7e2",
+      "metadata": {},
+      "source": [
+        "## Install Dependencies"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "id": "a41bc3e4-ea52-43aa-9239-a431b49f029e",
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "!pip install -qU vecs datasets llama_index html2text"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "0026437c",
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "import logging\n",
+        "import sys\n",
+        "\n",
+        "# Uncomment to see debug logs\n",
+        "# logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)\n",
+        "# logging.getLogger().addHandler(logging.StreamHandler(stream=sys.stdout))\n",
+        "\n",
+        "from llama_index import SimpleWebPageReader, StorageContext\n",
+        "from llama_index.indices.vector_store import VectorStoreIndex\n",
+        "from llama_index.vector_stores import SupabaseVectorStore\n",
+        "import textwrap\n",
+        "import html2text"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "id": "7a95a440",
+      "metadata": {},
+      "source": [
+        "# Set up OpenAI\n",
+        "\n",
+        "LlamaIndex uses OpenAI GPT-3 `text-davinci-003` model. Let's store that in an environment variable:"
```

Comment:
This model is slated for deprecation (shutdown in Jan), so might not be a good one to use for a long-term example.

Reply:
Thanks for the review @ted-at-openai, I've went ahead and removed the llamaindex example. PTAL.